sig/line/bot/v2/webhook_parser.rbs

```diff
@@ -5,8 +5,9 @@ module Line
         class InvalidSignatureError < ::StandardError
         end
         @channel_secret: String
+        @skip_signature_verification: Proc?
 
-        def initialize: (channel_secret: String) -> void
+        def initialize: (channel_secret: String, ?skip_signature_verification: Proc?) -> void
```

Comment:
Can we address this by writing like this, so as not to ignore in RBS? Type errors should be pointing out issues, so I'd like to avoid adding `# steep:ignore` as much as possible.

like 
```ruby
interface SkipSignatureValidationPredicate
  def call: () -> bool
end
...
@skip_signature_verification: SkipSignatureValidationPredicate?
```